### DISCUSSION TITLE
’Validation of the dislocation transport equations‘ by Christoph Kords, nonlocal

### DISCUSSION THREAD

**Discussion by cherry12306** [damask-multiphysics/DAMASK#13] [Q&A]
Hello, I am a beginner in damask and I am trying to reproduce chapter 5 Validation of the dislocation transport equations of Christoph Kords' PhD thesis. I am having 2 problems and I hope you can give me some guidance. First of all, regarding the dislocation source being at the origin and the positive edge dislocation moving in the positive direction of the x-axis, and the negative edge dislocation moving in the negative direction of the x-axis, the dislocation didn't move after I ran my model, I will attach my material.yaml file below, as well as a schematic of my model. Secondly, regarding the elastic matrix, I did not find the shear modulus and poisson ratio in the isotropic example, and I did not find the dislocation mobility in the nonlocal constitutive law. I will attach all my files below, and I hope I can get your answer. Thank you very much.
[fujian.zip](https://github.com/eisenforschung/DAMASK/files/15156738/fujian.zip)

**Comment by MarDiehl** [comment]
I don't recommend beginners to start with the nonlocal model because of its complexity.

**Reply by cherry12306** [reply]
Thank you, but my form of force requires me to use this constitutive law, I'll do my best to try again.

**Reply by MarDiehl** [reply]
Then I suggest that you start learning DAMASK with simpler constitutive models and switch to nonlocal only after you became a DAMASK professional.

**Reply by cherry12306** [reply]
Thanks for the suggestion, I've tried Phenomenological constitutive models before, and nonlocal is indeed a bit harder than the previous one. I need to learn more.
